go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/ianlancetaylor/demangle v0.0.0-20200824232613-28f6c0f3b639
+	github.com/orcaman/concurrent-map v1.0.0
```

Comment:
Is this a new dependency?

Reply:
Yes, this works to reduce lock contention during map operations in receiving the event from BPF. I will update the license before release.